### DISCUSSION TITLE
Idea: Enable scroll-to-zoom for images before presenting them

### DISCUSSION THREAD

**Discussion by ux-git** [sircharlo/meeting-media-manager#2302] [Ideas] [answered]
### Is your idea related to a problem?

Sometimes there are square/vertical images the can be easily zoomed in without loosing important details. Currently, I couldn't find a way to zoom in the image **before** presenting them. Doing this while presenting moves people attention from a brother to the image too much in my opinion.

### Describe your idea

It would be great if "Scroll-to-zoom" feature will be accessible even before we present the image. So once presented – the image already fits the aspect ratio and looks good.

### Describe alternatives you've considered

Zooming while presenting the image, cropping an image in advance (too much extra efforts).

**Comment by e-Feitosa** [comment]
Good suggestion, I was going to suggest the same thing for the option to access the jw.org website. Being able to have the option to show after clicking on the option and if disabled, it would not show the screen.

**Reply by ux-git** [reply]
Great idea! This will also be useful. For example, open the website and have a big blue button "Present" to stream the video to a second monitor. Then the text on the button can be changed to "Stop presenting" to stop showing it. But the overall flow is currently quite good.

**Comment by mtdvlpr** [comment]
Present in #2781

**Comment by sircharlo** [comment, marked as answer]
Implemented in 21d0a9930ec88fba65a33872cce408a46a291795. Will be released in the next update!
